plugins/remote-config/frontend/public/templates/parameters.html

```diff
@@ -1,36 +1,88 @@
 <div>
-    <cly-header title="Parameters">
-        <template v-slot:header-right>
-            <div class="bu-level-item">
-                <el-button @click="create" type="primary" size="small" icon="el-icon-circle-plus">Add Parameter</el-button>
+  <cly-header>
+    <template v-slot:header-left>
+      <h2 class="cly-vue-remote-config__header"> {{i18n('remote-config.title')}} 
+      </h2>
+      <cly-tooltip-icon icon="ion-help-circled"></cly-tooltip-icon>
+    </template>
+    <template v-slot:header-right>
+      <div class="bu-level-item">
+        <el-button @click="create" type="primary" size="small" icon="el-icon-circle-plus">{{i18n('remote-config.add-parameter-title')}}</el-button>
+      </div>
+    </template>
+  </cly-header>
+  <cly-main>
+    <json-editor></json-editor>
+    <cly-datatable-n :rows="tableRows">
+      <template v-slot:default="scope">
+        <el-table-column type="expand" min-width="50">
+          <template v-slot="props">
+            <cly-section>
+              <el-tabs type="border-card" value="default_value">
+                <el-tab-pane name="default_value" :label="i18n('remote-config.default-value')">
+                  <div>
+                    <pre>{{ props.row.default_value }}</pre>
+                  </div>
+                </el-tab-pane>
+                <el-tab-pane name="conditions" :label="i18n('remote-config.conditions')">
```

Comment:
this tab should only be shown if drill is enabled. please add the check for that.

Reply:
yes right, will add it here:)